### PR TITLE
Add credits to the KaTeX team and contributors

### PR DESCRIPTION
**Pull request by ylemkimon**
* It is possible to [change the visibility in the team page](https://help.github.com/articles/publicizing-or-hiding-organization-membership/)
* It'd be nice to have a Team/Contributors page like:
  * https://babeljs.io/team
  * https://angular.io/about?group=Angular
  * https://jquery.org/team/

### CODE REVIEW

**Comment by kevinbarabash** [comment]
@ylemkimon yes!  Thank you for adding this.

**Comment by kevinbarabash** [comment]
The "team" link is maybe a little confusing b/c not everyone who has push access to `khan/KaTeX` has been added to the `KaTeX` organization.  Let's keep the "Contributors" link for now.  Hopefully we can get the repo moved over to the `KaTeX` organization eventually and this will be less confusing.  I like how the babel page differentiates core members as well as active vs. non-active members.

File: website/pages/index.html [hunk 1]
```diff
@@ -114,7 +114,8 @@ <h1 class="logo tex" data-expr="\KaTeX">KaTeX</h1>
     <div class="footer"><div class="container">
       <a class="ka-logo" href="https://www.khanacademy.org/"><img src="img/khan-academy.png" alt="Khan Academy" width="180" height="17"></a>
       <span class="credits">
-        Created by <a href="https://github.com/xymostech">Emily Eisenberg</a> and <a href="https://sophiebits.com/">Sophie Alpert</a>
+        Created by <a href="https://github.com/xymostech">Emily Eisenberg</a> and <a href="https://sophiebits.com/">Sophie Alpert</a><br>
+        Maintained by <a href="https://github.com/orgs/KaTeX/people">the KaTeX team</a> with help from <a href="https://github.com/Khan/KaTeX/graphs/contributors">contributors</a>
```
Comment (kevinbarabash) [inline website/pages/index.html]: Nice!

File: website/core/Footer.js [hunk 2]
```diff
@@ -64,6 +64,8 @@ class Footer extends React.Component {
               </div>
               <div>
                 <h5>More</h5>
+                <a href="https://github.com/orgs/KaTeX/people">Team</a>
```
Comment (kevinbarabash) [inline website/core/Footer.js]: Leave this out.

File: website/pages/index.html (hunk 1 above)
Comment (ylemkimon) [reply website/pages/index.html]: @kevinbarabash This also includes the link to KaTeX organization. Should I remove it?

Comment (kevinbarabash) [reply website/pages/index.html]: I didn't notice that.  Yeah, let's remove that as well.

Comment (ylemkimon) [reply website/pages/index.html]: @kevinbarabash Could you suggest how should I write it?

Comment (ylemkimon) [reply website/pages/index.html]: What do you think of `KaTeX was built from the hard work of all these contributors.`

Comment (edemaine) [reply website/pages/index.html]: Sounds good!  Slight tweak: Maybe `... hard work of [many contributors](https://github.com/Khan/KaTeX/graphs/contributors)`. I think it'd be good to keep the "created by" line as well.  Maybe `KaTeX was built from the hard work ...`  It was originally created by ...`?

**Comment by ylemkimon** [comment]
Also, I'm thinking to adding license information to the footer:

image

Any opinions?

File: website/pages/index.html (hunk 1 above)
Comment (ylemkimon) [reply website/pages/index.html]: @edemaine I was just thinking to append the new line after existing line.

**Comment by sophiebits** [comment]
I'm very, very late here, but thank you for adding this!

**Pull request merged**